INFO     eawag.datapool  connected to db sqlite+pysqlite:///<tmpdir_from_fixture>/datapool.db
INFO     eawag.datapool  connected to db sqlite+pysqlite:///<tmpdir_from_fixture>/datapool.db
INFO     eawag.datapool  created all tables of db sqlite+pysqlite:///<tmpdir_from_fixture>/datapool.db
INFO     eawag.datapool  use parse_site to parse all files from <tmpdir_from_fixture>/landing_zone
INFO     eawag.datapool  load <tmpdir_from_fixture>/landing_zone/sites/example_site/site.yaml
DEBUG    eawag.datapool  called check for site test_site
INFO     eawag.datapool  called commit for site test_site
DEBUG    eawag.datapool  add new site row <SiteDbo name=test_site, city=Schacht 24G3S, Mischsystem, x=580000, y=255300, z=455.3, description=All the details of the site.All the details the site.All the details of the site.All the details of the site.All the details of the site.All the details of the site.All the details of the site.>
<SiteDbo name=test_site, city=Schacht 24G3S, Mischsystem, x=580000, y=255300, z=455.3, description=All the details of the site.All the details the site.All the details of the site.All the details of the site.All the details of the site.All the details of the site.All the details of the site.>
INFO     eawag.datapool  use parse_site to parse all files from <tmpdir_from_fixture>/landing_zone
INFO     eawag.datapool  load <tmpdir_from_fixture>/landing_zone/sites/example_site/site.yaml
DEBUG    eawag.datapool  called check for site test_site
DEBUG    eawag.datapool  site with same name <SiteDbo name=test_site, city=Schacht 24G3S, Mischsystem, x=580000, y=255300, z=455.3, description=All the details of the site.All the details the site.All the details of the site.All the details of the site.All the details of the site.All the details of the site.All the details of the site.> already exists
ERROR    eawag.datapool  detected not allowd modifications of pictures in landing zone:
ERROR    eawag.datapool  existing picture:
ERROR    eawag.datapool  PictureDbo:
ERROR    eawag.datapool      filename    : '1.jpg'
ERROR    eawag.datapool      timestamp   : datetime.datetime(2016, 1, 1, 0, 0)
ERROR    eawag.datapool      filesize    : 356
ERROR    eawag.datapool      description : 'rainevent'
ERROR    eawag.datapool  modified picture:
ERROR    eawag.datapool  Picture:
ERROR    eawag.datapool      filename    : '1.jpg'
ERROR    eawag.datapool      timestamp   : datetime.datetime(2016, 1, 1, 10, 0)
ERROR    eawag.datapool      filesize    : 356
ERROR    eawag.datapool      description : 'rainevent'

check for site update failed, see logs for details

